styles/ContactUs.module.css

```diff
@@ -0,0 +1,70 @@
+.title {
+  text-align: center;
+  color: var(--primary-blue);
+  padding: var(--padding-lg);
+}
+.subtitle {
+  color: var(--primary-blue);
+}
+.subtitleBasedIn {
+  color: var(--primary-blue);
+  padding-bottom: 17.93px;
+}
+.mail {
+  text-decoration: none;
+  color: var(--primary-blue);
+  font-size: 1.25rem;
+}
+.mailimg {
+  visibility: hidden;
+}
+.address {
+  padding-top: 41px;
```

Comment:
Please use padding vars on the rest of the file

Reply:
done